reports/uds_log_report.txt

```diff
@@ -20,3 +20,23 @@ Requested Reset Initiated: Yes
 Suppress Positive Message Request: False
 <------------------- LOG ENTRY END ------------------->
 
+<---- LOG ENTRY [ass930085 - 2024-10-21 14:15:39] ---->
+UDS Request :   [0x85 0x1]
+Explaination:   Control DTC Setting (Service 85) Requested for Control DTC Setting type 0x1 ON and SPRMIB flag False
```

Comment:
The explaination must be updated. if ON is requested then statement must be Request to Enable setting of DTCs and for OFF it must be Requested to diable setting the DTCs. Similarly update the statements in response as well. It must be human readable. Usually users may not know what does subfunction 1 or 2 mean ad ON and OFF mea right.